task_sdk/tests/execution_time/test_supervisor.py

```diff
@@ -689,6 +690,27 @@ def watched_subprocess(self, mocker):
                 "",
                 id="patch_task_instance_to_deferred",
             ),
+            pytest.param(
+                SetXCom(
+                    dag_id="test_dag",
+                    run_id="test_run",
+                    task_id="test_task",
+                    key="test_key",
+                    value='{"key": "test_key", "value": {"key2": "value2"}}',
+                ),
+                b"",
+                "xcoms.set",
+                (
+                    "test_dag",
+                    "test_run",
+                    "test_task",
+                    "test_key",
+                    '{"key": "test_key", "value": {"key2": "value2"}}',
+                    None,
+                ),
+                {"message": "XCom successfully set"},
```

Comment:
This needs changing, now that you use `{"ok": True}`

Reply:
Changed